tests/fixtures/ecma-version/13/regexp-match-indices/regexp-match-indices.result.js

```diff
@@ -0,0 +1,199 @@
+import conditionalRegexValue from "../../../../lib/conditional-regex-value.js";
+import conditionalRegex from "../../../../lib/conditional-regex-value.js";
```

Comment:
```suggestion
import conditionalRegex from "../../../../lib/conditional-regex-value.js";
```

One is a duplicate. Other similar tests are using `conditionalRegex` name.